src/lib/y2partitioner/actions/controllers/md.rb

```diff
@@ -249,8 +249,7 @@ def available?(partition)
           return false if partition.md
           return true if partition.filesystem.nil?
 
-          mount = partition.filesystem.mountpoint
-          mount.nil? || mount.empty?
+          partition.filesystem.mount_point.nil?
```

Comment:
Same as above: Can there be a mount point, but with a nil/empty path?

Reply:
The same applies here.